fesvr/htif.h

```diff
@@ -26,7 +26,7 @@ class htif_t : public chunked_memif_t
   int run();
   bool done();
   int exit_code();
-
+  virtual void set_max_xlen(unsigned int m) {max_xlen = m;};
```

Comment:
No need for `virtual`, or the last semicolon

Reply:
Oops! Fixed.